quality-assurance/Security-Testing-Guidance.md

```diff
@@ -0,0 +1,116 @@
+# Overview
+This document is intended to provide some guidance as to what level of testing should be considered within the project when 
+tackling security testing, and is aimed at the Shift Left approach to test early. This will look at the OWASP as a source of 
+the test area that needs to be covered.
```

Comment:
```suggestion
This document provides guidance as to what level of testing should be considered within a project when 
tackling security testing, and is aimed at the Shift-Left approach to test early. This will look at OWASP as a source of 
test areas that need to be covered.
```